stdlib/builtins.pyi

```diff
@@ -226,7 +229,7 @@ class int:
     def __trunc__(self) -> int: ...
     def __ceil__(self) -> int: ...
     def __floor__(self) -> int: ...
-    def __round__(self, ndigits: Optional[int] = ...) -> int: ...
+    def __round__(self, ndigits: SupportsIndex | None = ...) -> int: ...
```

Comment:
`None` doesn't seem to be right:

```python
>>> 1 .__round__(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object cannot be interpreted as an integer
```

Reply:
You're right, fixed as of https://github.com/python/typeshed/pull/5739/commits/17210c3183acf0d5cc69b50d0d83be062d46dd13.